src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -282,7 +316,12 @@ export function DashboardTopNav({ embedSettings, redirectTo }: DashboardTopNavPr
   }, [hasUnsavedChanges, viewMode, hasRunMigrations, showWriteControls, managed]);
 
   return (
-    <div className="dashboardTopNav">
+    <div
+      className={classNames('dashboardTopNav', {
+        'dashboardTopNav-noBackgroundColor': !visibilityProps.showBackgroundColor,
+        'dashboardTopNav-noSticky': !visibilityProps.showStickyTopNav,
```

Comment:
Regarding `wrapperCss`, and the `visibilityProps`. I'm not sure if these types of props are appropriate. Maybe we need to revisit how the dashboard top nav is broken up in order to allow the stickiness and the background color to be dictated by whichever component renders the top nav, instead of being props passed in.

Reply:
Have these removed after aligning the styles with analytic dashboard.